rclrs/CHANGELOG.rst

```diff
@@ -5,11 +5,13 @@ Changelog for package rclrs
 0.3
 ----------------
 * Loaned messages (zero-copy) (`#212 <https://github.com/ros2-rust/ros2_rust/pull/212>`_)
+* Graph support (`#234 <https://github.com/ros2-rust/ros2_rust/pull/234>`_)
```

Comment:
Maybe this could be `Graph queries` instead of `Graph support`, since there is also the graph listener in rclcpp which we don't have yet.

Reply:
I agree about graph queries, graph support basically includes that and things like `wait_for_service`, which is not implemented yet.